internal/backuprestore/restore.go

```diff
@@ -232,7 +232,7 @@ func (h *BRHandler) LoadRestoresFromOadpRestorePath() ([][]*velerov1.Restore, er
 
 		// The returned list of entries are sorted by name alphabetically
 		restoreDirPath := filepath.Join(OadpRestorePath, restoreSubDir.Name())
-		restoreYamls, err := os.ReadDir(filepath.Join(hostPath, restoreDirPath))
+		restoreYamls, err := os.ReadDir(filepath.Clean(filepath.Join(hostPath, restoreDirPath)))
```

Comment:
Too bad this isn't reported by golangci-lint / gosec.

Reply:
Yeah pretty strange